src/mlt++/MltAnimation.cpp

```diff
@@ -101,6 +101,15 @@ mlt_keyframe_type Animation::keyframe_type( int position )
 		return (mlt_keyframe_type) -1;
 }
 
+/** Get the keyfame at the position or the next following.
```

Comment:
We do not maintain doxygen comments for the C++ wrapper BTW. You can keep it because you documented as deprecated, but I have no intention to add doxygen comments throughout.

Reply:
I understand. But these functions to not map directly to the underlying C functions that they use. And I wanted the intent/design to be documented somewhere. Also, I wanted to mark them deprecated so that they show up the next time I search for "deprecated" functions to remove.

I also do not intend to add doxygen comments to the entire C++ API.